apis/v1beta1/config.go

```diff
@@ -282,6 +283,10 @@ func (c *Config) GetExporterPorts(logger logr.Logger) ([]corev1.ServicePort, err
 	return c.getPortsForComponentKinds(logger, KindExporter)
 }
 
+func (c *Config) GetExtensionPorts(logger logr.Logger) ([]corev1.ServicePort, error) {
+	return c.getPortsForComponentKinds(logger, KindExtension)
+}
```

Comment:
note: not all extensions create extra servers, an extensions could be just a library for use elsewhere, e.g. encoding extensions, or even jaeger storage.

Reply:
It's fine. There is a mechanism to register those components that provide a port or something special like RBAC.